ng-dev/pr/merge/merge-tool.ts

```diff
@@ -143,8 +135,26 @@ export class MergeTool {
       // Run preparations for the merge (e.g. fetching branches).
       await strategy.prepare(pullRequest);
 
+      // Check for conflicts between the pull request and target branches.
+      await strategy.check(pullRequest);
+
+      if (this.flags.dryRun) {
+        Log.info(green(`  ✓  Dry run sucessful, pull request merges as expected.`));
```

Comment:
In the prompt IIRC we print the target branches. If this is true, we should move this out and print it in a single place, even with dry run mode

Reply:
Might be worth always stating that the PR merges cleanly, and just exiting early with dry-mode. That way it would be clear before the prompt that the mergability has already been checked

Reply:
I updated the message as suggested, though they will not see the prompt because we `return` out of the method before we continue on to it.